src/usbDetector.ts

```diff
@@ -21,8 +21,9 @@ export interface DeviceInfo {
 export class UsbDetector {
   private static _vscexpress: VSCExpress|undefined;
   // tslint:disable-next-line: no-any
-  private static _usbDetector: any =
-      require('../../vendor/node-usb-native').detector;
+  private static _usbDetector: any; 

```

Comment:
hmm.. should this file  be remove from the workbench and move to the helper extension?

Reply:
Yes. This part of code should be removed and move to the helper extension. Here I comment out the code because the useDetector's callback function is blocking the VS Remote from connecting back to local window.
Code related to uploading file to device is supposed to be cleaned up.